### DISCUSSION TITLE
[vNext] Add option to 'Publish' instead of deploy

### DISCUSSION THREAD

**Discussion by DamianSuess** [SuessLabs/VsLinuxDebug#86] [General]
Publish project option

**Comment by DamianSuess** [comment]
Deprecating feature

**Comment by Reveon** [comment]
Having a publish option would be great for ASP.NET apps. For example, in my case, I have a Blazor WebAssembly app hosted on ASP.NET. After building and deploying, the 'wwwroot' folder from my Blazor project isn't transferred to the Raspberry Pi since build itself doesn't account for that folder.

Currently, the only way to make this work is to publish the app, take the 'wwwroot' folder from the Blazor Wasm project, copy it to the debug folder in the host ASP.NET project, and then run the Build and Deploy.

Otherwise great work!

**Comment by DamianSuess** [comment]
That's good to know someone would find use in that feature. It's in there (_from what I recall_) but partially commented out and untested.

If this is something you would find useful, we could add it.

**Comment by Reveon** [comment]
If it's not too much effort for you to finish it, then yes, it would be great to have that.

**Comment by DamianSuess** [comment]
Let's open it up and try to get it in for the 2023-Q4 release.
This upcoming release aims to support multiple connection profiles. So it should be able to make the cut.
